src/axom/sidre/core/MFEMSidreDataCollection.cpp

```diff
@@ -1008,8 +1043,37 @@ void MFEMSidreDataCollection::Save(const std::string& filename,
   if(m_comm != MPI_COMM_NULL)
   {
     IOManager writer(m_comm);
-    sidre::DataStore* datastore = m_bp_grp->getDataStore();
-    writer.write(datastore->getRoot(), num_procs, file_path, protocol);
+
+    // Create a shallow mirror of the DataStore structure that only includes
```

Comment:
Does this only output the buffers associated with the current mesh? Or will all buffers be dumped?

Reply:
Regardless of whether MFEMSidreDataColl owns its data the `named_buffers` group exists within the (mesh-specific) domain group:

https://github.com/LLNL/axom/blob/8478a6f3552f68af66bb11ae18a0d516d2d35305/src/axom/sidre/core/MFEMSidreDataCollection.cpp#L101

https://github.com/LLNL/axom/blob/8478a6f3552f68af66bb11ae18a0d516d2d35305/src/axom/sidre/core/MFEMSidreDataCollection.cpp#L134

Accordingly I think only the (named) buffers used by the domain and global groups are saved - additionally this method here

https://github.com/LLNL/axom/blob/8478a6f3552f68af66bb11ae18a0d516d2d35305/src/axom/sidre/core/Group.cpp#L1956-L1958

gets used internally (a couple layers down) to identify the `Buffer`s (managed by the datastore itself) associated with those domain and global groups.

Reply:
That's what I thought (re: `exportTo`).  
Would it make sense to confirm that this is the case?

Reply:
Hmm, I'm not sure what the best way to check this would be since I didn't see any Group methods that report the corresponding buffers (the one used here is `private`), though perhaps I'm just missing something

Reply:
We don't have that built-in.
I was thinking more about manually checking/confirming, e.g. by adding a really large buffer and seeing if this affected the output size.

Reply:
I've gone ahead and added an an extraneous View to the datastore and later check that it's not reloaded, which I believe confirms the same thing